packages/alert/src/Alert.js

```diff
@@ -16,15 +16,17 @@ class Alert extends React.Component {
     autoRemove: PropTypes.bool,
     onDismiss: PropTypes.func,
     visibleTime: PropTypes.number,
-    zIndex: PropTypes.number
+    zIndex: PropTypes.number,
+    bottomPosition: PropTypes.bool
```

Comment:
I guess we could name this property `alignTo` and it would receive `top` or `bottom` with its default value set to `top`. 
Or even 'alignToBottom'...
Thoughts?

Reply:
I think I like `alignToBottom` better, as a boolean prop. It will be rarely used and, when it happens, it will be little bit more concise.

Reply:
bottomAligned maybe

Reply:
bottomAligned is good too. I'll go with it